ntbs-integration-tests/Helpers/TestDocumentExtensions.cs

```diff
@@ -13,13 +15,35 @@ public static string GetError(this IHtmlDocument document, string input)
             return errorSpan.TextContent;
         }
 
+        public static string GetErrorFromSummary(this IHtmlDocument document, string errorSummaryName, string errorSpanName)

```

Comment:
Added a new method to get the error from the Summary that links to where the actual error is contained rather than searching the whole document for it - will start replacing all applicable tests with this